### PR TITLE
adding nutritionits list and table component

### LINKED ISSUE #20: create nutritionists list component

### PR DESCRIPTION
**Pull request by mnzgarcia**
Fix #20

### CODE REVIEW

**Review by AdanRL — APPROVED** [review]

**Review by AdanRL — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ function App() {
       </BrowserRouter>
       <Footer />
       <SignUp />
+      <NutritionistList />
```
Comment (elenamsd) [inline src/App.tsx:41]:
```suggestion
```

File: src/App.tsx [hunk 2]
```diff
@@ -37,6 +38,7 @@ function App() {
       </BrowserRouter>
       <Footer />
       <SignUp />
```
Comment (elenamsd) [inline src/App.tsx:40]:
```suggestion
```

**Review by AdanRL — APPROVED** [review]

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  "&:nth-of-type(odd)": {
+    backgroundColor: theme.palette.action.hover,
+  },
+  // hide last border
```
Comment (elenamsd) [inline src/components/Table/Table.tsx:38]:
```suggestion
```

File: src/components/Table/Table.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+export const StickyHeadTable: FC<Props> = ({ titles, data }) => {
+  const columns: readonly Column[] = titles.map((title) => {
+    return { id: title.id, label: title.label, minWidth: title.minWidth }; 
+  });
+
+  const rows: Data[] = data.map((item) => {
+    return { name: item.name, specialties: item.specialties, contact: item.email, calendar: item.username };
+  });
+
+  console.log(data);
```
Comment (elenamsd) [inline src/components/Table/Table.tsx:53]:
```suggestion
```

File: src/components/Table/Table.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+export const StickyHeadTable: FC<Props> = ({ titles, data }) => {
+  const columns: readonly Column[] = titles.map((title) => {
+    return { id: title.id, label: title.label, minWidth: title.minWidth }; 
+  });
+
+  const rows: Data[] = data.map((item) => {
+    return { name: item.name, specialties: item.specialties, contact: item.email, calendar: item.username };
+  });
+
```
Comment (elenamsd) [inline src/components/Table/Table.tsx:52]:
```suggestion
```

**Review by elenamsd — CHANGES_REQUESTED** [review]
🧐

File: src/App.tsx [hunk 6]
```diff
@@ -9,6 +9,7 @@ import { Page404 } from "./pages/Page404/Page404";
 import { Footer } from "./components/Footer/Footer";
 import { NavBar } from "./components/NavBar/NavBar";
 import { SignUp } from "./components/SignUp/SignUp";
+import { NutritionistList } from "./components/NutritionistList/NutritionistList";
```
Comment (elenamsd) [inline src/App.tsx:12]:
```suggestion
```

**Review by elenamsd — CHANGES_REQUESTED** [review]
🧐

**Review by elenamsd — APPROVED** [review]

**Pull request merged**